tutorials/beginner/installing.md

```diff
@@ -1,27 +1,55 @@
 ---
 layout: tutorial-beginner
+title: Installing Extension
 ---
 
 {% include base.html %}
 
-# To be done
+# Installing an Extension
```

Comment:
To be consistent, "Extensions" is not the word you should use. [Add-ons](http://docs.openhab.org/addons/index.html)